src/appkit.cs

```diff
@@ -5748,6 +5799,11 @@ partial interface NSDocument /* : NSUserActivityRestoring radar://42781537 */ {
 		[Export ("restorableStateKeyPaths", ArgumentSemantic.Copy)]
 		string [] RestorableStateKeyPaths ();
 
+		[Mac (12,0)]
+		[Static]
+		[Export ("allowedClassesForRestorableStateKeyPath:")]
+		Class[] AllowedClassesForRestorableStateKeyPath (string keyPath);
```

Comment:
Methods should have a verb:

```suggestion
		Class[] GetAllowedClassesForRestorableStateKeyPath (string keyPath);
```